### ISSUE TITLE
WARN 'more than one version...' printed when there is only one version of a resource but two copies.

### ISSUE THREAD

**Issue by mjoaquim12** [l0rdn1kk0n/wicket-webjars#63]
See https://github.com/snapappointments/bootstrap-select

META-INF/resources/webjars/bootstrap-select/1.13.12/docs/docs/dist/css/bootstrap-select.css
META-INF/resources/webjars/bootstrap-select/1.13.12/dist/css/bootstrap-select.css

WARN  2020 Apr 19, 15:05:12 [localhost-startStop-1] WicketWebjars - more than one version of a dependency is not supported till now. webjars resource: /dist/css/bootstrap-select.css; available versions: [1.13.12, 1.13.12]; using: 1.13.12

Also prints the same warning for bootstrap-select.js.

**Comment by martin-g** [comment]
What is the complaint here ?
There are two `css/bootstrap-select.css` here. It is a wrong packaging of this particular webjar.

**Comment by mjoaquim12** [comment]
The path provided in code is
new WebjarsCssResourceReference( "bootstrap-select/current/dist/css/bootstrap-select.css" ).

Not sure why the /docs directory is being scanned. 

Thank you.

**Comment by bgooren** [comment]
We keep seeing this warning as well. Many webjars have this issue, since they embed source code and "compiled"/bundled + minified code.

I think it's confusing for users, since the warning lists the same version number many times (e.g. for one of the libraries we use (moment.js), it logs the same version number 4 times), and says "more than one version is not supported". But all the version numbers are the same.

Perusing the source code I see that all the contents of a webjar are scanned. So I understand where the warning comes from.

But practically speaking it's not ever been an issue for us (like @mjoaquim12), since we reference a folder like "dist".

Perhaps we should de-dup the list of versions, and if there actually is more than one distinct version, log the warning? I can make a pull request for that if you agree that it makes sense.

**Comment by martin-g** [comment]
PRs and/or tests and/or quickstarts are welcome!

Nothing personal but creating an issue without providing a test/quickstart/steps to reproduce will most likely be ignored like here...

**Comment by bgooren** [comment]
I have created a quickstart to demonstrate the issue.

[myproject.zip](https://github.com/l0rdn1kk0n/wicket-webjars/files/12584170/myproject.zip)

If you start it, and simply access the homepage (http://localhost:8080/), you will see the following warning logged:

```
[qtp762227630-15] WARN de.agilecoders.wicket.webjars.WicketWebjars - more than one version of a dependency is not supported till now. webjars resource: /moment.js; available versions: [2.29.4, 2.29.4, 2.29.4, 2.29.4]; using: 2.29.4
```

Granted, the webjar that causes this is a bit messy (it contains the javascript reference 4 times). But the warning that more than one version is found, with all versions being equal, is rather confusing.
And in the end, as long as I reference the correct path (in this case either /moment/current/moment.js, /moment/current/dist/moment.js or /moment/current/min/moment.min.js), everything works just fine.

**Comment by martin-g** [comment]
I've applied a simple fix which should be enough for now!
Thank you for the quickstart!
